manifest.base.yml

```diff
@@ -1,13 +1,14 @@
 ---
 buildpack: python_buildpack
-command: python manage.py migrate --fake-initial && waitress-serve --url-prefix /regulations --port=$VCAP_APP_PORT fec_eregs.wsgi:application
+command: python manage.py migrate --fake-initial && gunicorn fec_eregs.wsgi:application
```

Comment:
I mistakenly followed suit with ATF eRegs, EPA notice-and-comment, etc in not explicitly specifying the port and letting it default to 8000. Your comment made me realize that it is probably an incorrect assumption and I'll push a new commit making the app explicitly use the port specified by the CloudFoundry runtime.